scylla/Cargo.toml

```diff
@@ -39,6 +39,7 @@ full-serialization = [
     "num-bigint-04",
     "bigdecimal-04",
 ]
+cpp-rust = []
 
 [dependencies]
 scylla-macros = { version = "0.7.0", path = "../scylla-macros" }
```

Comment:
1. Can you make it a cfg instead of a feature? This is to make it harder to accidentaly enable, e.g. by some dependency. Tokio does it the same way: https://docs.rs/tokio/latest/tokio/#unstable-features
2. Please prepend "unstable-" to the name - this is a popular convention to mark that contents of this feature / cfg do not adhere to semver.

Reply:
Great suggestion. I applied it (but I appended the `unstable` suffix :)).